task/github-create-deployment/0.1/README.md

```diff
@@ -0,0 +1,66 @@
+# GitHub
+
+A collection of tasks to help working with the [GitHub
+API](https://developer.github.com/v3/).
+
+## GitHub token
+
+Most tasks would expect to have a secret set in the kubernetes secret `github`
```

Comment:
```suggestion
This task expect to have a secret set in the kubernetes secret `github`
```

Reply:
(probably need to fix the wording of the other github tasks* who got moved to their own directory)